packages/buefy-next/src/components/rate/Rate.spec.ts

```diff
@@ -52,7 +54,7 @@ describe('BRate', () => {
     it('manage methods correctly', async () => {
         const event = {
             stopPropagation: jest.fn()
-        }
+        } as unknown as Event
```

Comment:
@kikuomax I feel like this is a messy way to get TS not to complain about this constant

Is there a cleaner method for this?

Reply:
Please see [this comment](https://github.com/ntohq/buefy-next/pull/277/files#r1759722041).